content/news/2021-XX-XX-fsf-statement.md

```diff
@@ -0,0 +1,24 @@
+title: "A Statement on Richard Stallman and the Free Software Foundation"
+authors: Owen Williams
+tags: community, diversity, politics
+date: 2021-03-26 00:00:00
+
+_A Statement from Mixxx on Richard Stallman's return to the FSF._
```

Comment:
This is redundant with the title of the blog post. I think it should be removed.
![image](https://user-images.githubusercontent.com/9455094/112706509-a1fc1400-8e72-11eb-855c-3c0a869616d8.png)